### DISCUSSION TITLE
How could I modify the location of databases by default?

### DISCUSSION THREAD

**Discussion by lee2015new** [orientechnologies/orientdb#9829] [Q&A] [answered]
Hi,
I installed orientdb by default in path /opt/orientdb recently. But my system hard disk only has 50G storage space. I want to change the default data saving path from ${ORIENTDB_HOME}/databases to another path. But I couldn't find the solution from website. Could anybody teach me how to reset the path of storage. Thanks very much.
By the way , my orientdb version is 3.1.11 and installed in ubuntu.

**Comment by lee2015new** [comment, marked as answer]
Can anybody help?

**Reply by tglman** [reply]
you can use the `server.database.path` property in the server xml

**Reply by lee2015new** [reply]
Thanks very much
